### PR TITLE
2009 fix month inputs

### PR DESCRIPTION
**Pull request by it-harrison**
## Chromatic

https://2009-fix-month-inputs--60f9b557105290003b387cd5.chromatic.com

---

## Description
This PR harmonizes the size of the month input (both the input text box and the select input) in the v3 va-memorable-date component to the Sketch designs.

Closes https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2009

## Testing done
local testing on Chrome, Firefox, Safari, Edge, iOS simulator

### month uses `input`
Screenshot 2023-08-24 at 3 59 53 PM

### month uses `select` (month-select)

Simulator Screenshot - iPhone SE (3rd generation) - 2023-08-25 at 09 26 56

## Acceptance criteria
- [x] the sizes of the v3 month select/input are the same as in the Sketch designs

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Comment by powellkerry** [comment]
Looks like there is an issue on screen sizes under 480px
Screenshot 2023-08-25 at 10 59 21 AM

**Comment by it-harrison** [comment]
> Looks like there is a spacing issue on screen sizes under 480px Screenshot 2023-08-25 at 10 59 21 AM

@powellkerry good catch - fixed

Screenshot 2023-08-25 at 12 34 07 PM

File: packages/web-components/src/components/va-memorable-date/va-memorable-date.scss [hunk 1]
```diff
@@ -17,15 +17,33 @@
   margin-top: 16px;
 }
 
+.uswds-usa-form-group--month {
+  margin-right: 16px;
+}
+
+.uswds-usa-form-group--month-input {
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-memorable-date/va-memorable-date.scss:24]: Is this a specific USWDS class from one of their modules? Using the same prefix that they use is confusing to me if it's not one of theirs.

Comment (it-harrison) [reply packages/web-components/src/components/va-memorable-date/va-memorable-date.scss]: @jamigibbs good call - I have updated the names

Comment (jamigibbs) [reply packages/web-components/src/components/va-memorable-date/va-memorable-date.scss]: I guess I'm just wondering why did we need to change the name of this class to begin with?

Now I see `usa-form-group--month-input` and `v3-usa-form-group--month-input` both of which are only used in the v3 variation. And `usa-form-group--month-input` isn't even used in the markup anywhere now.

Comment (jamigibbs) [reply packages/web-components/src/components/va-memorable-date/va-memorable-date.scss]: Can we just use `usa-form-group--month-input` like it was and just adjust the CSS for it?

Comment (it-harrison) [reply packages/web-components/src/components/va-memorable-date/va-memorable-date.scss]: @jamigibbs PR has been updated to use only original classes - thanks for the nudge

File: packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx [hunk 2]
```diff
@@ -177,7 +177,9 @@ export class VaMemorableDate {
     const target = event.target as HTMLInputElement;
 
     let [currentYear, currentMonth, currentDay] = (this.value || '').split('-');
-    if (target.classList.contains('input-month') || target.classList.contains('usa-form-group--month-input') || target.classList.contains('usa-form-group--month-select')) {
+    if (target.classList.contains('input-month') || target.classList.contains('usa-form-group--month-input')
+      || target.classList.contains('usa-form-group--month-select')
+      || target.classList.contains('usa-form-group--month-input')) {
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-memorable-date/va-memorable-date.tsx:182]: Everything looks good! 

But am I reading this correctly that `target.classList.contains('usa-form-group--month-input')` is added twice? I'm not sure why we would need to add it again to this condition.

**Review by jamigibbs — APPROVED** [review]
Thank you! 👍 Before merging, can you:

- Update the PR title so that it includes the name of the component (for our release notes). ie. va-memorable-date: Update month input width
- Increment the web-components package.json version

**Pull request merged**